src/test/java/lexi/task/TodoTest.java

```diff
@@ -0,0 +1,49 @@
+package lexi.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Should the statements follow a consistent format? There is a line separator for your other files. (Just a nit)
```suggestion
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;
```